### PR TITLE
[server][da-vinci] reduced DIV warn logging frequency to reduce cpu wait especially during rewind

### PR DESCRIPTION
**Pull request by gaojieliu**
## Summary, imperative, start upper case, don't end with a period
reduced DIV warn logging frequency to reduce cpu wait especially during rewind

## How was this PR tested?
CI

## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

File: clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java [hunk 1]
```diff
@@ -2777,12 +2777,16 @@ protected void validateMessage(
 
       // TODO: remove this condition check after fixing the bug that drainer in leaders is validating RT DIV info
       if (consumerRecord.getValue().producerMetadata.messageSequenceNumber != 1) {
-        LOGGER.warn(
-            "Data integrity validation problem with: {} offset: {}, "
-                + "but consumption will continue since EOP is already received. Msg: {}",
-            consumerRecord.getTopicPartition(),
-            consumerRecord.getOffset(),
-            warningException.getMessage());
+        String errorMsgIdentifier = consumerRecord.getTopicPartition().getPubSubTopic().getName() + "-"
```
Comment (sixpluszero) [inline clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java:2780]: but warnMessage should contain details of producer right? Should we ignore that?

Comment (sixpluszero) [reply clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java:2780]: NVM. Didn't see the getClass().getName() this should be OK.

**Review by FelixGV — CHANGES_REQUESTED** [review]
I would prefer that we do something a bit different...

Specifically for the case of the non-SOS first message, completely suppress the log (and probably stop incrementing the missing metric as well, as it is misleading...), basically, treat it as a fully ok write (assuming EOP is already gotten of course).

For all other missing data, log it everytime. No redudant filter anywhere...

**Comment by gaojieliu** [comment]
This PR needs some re-work and I will take another look soon.

**Review by majisourav99 — APPROVED** [review]

**Comment by gaojieliu** [comment]
@lluwm has already merged a fix:
https://github.com/linkedin/venice/pull/905
I will discard this one.

**Pull request closed without merging**